sln/OData.Tests.Performance.md

```diff
@@ -0,0 +1,5 @@
+1. Install Visual Studio 2019 Enterprise 16.11 configuring the installer with the `.vsconfig` file [here](./OData.Tests.Performance.vsconfig) (other versions and SKUs of Visual Studio 2019 should also work, but these instructions were tested with the version specified here). Continue through any prompts about components that are out-of-support. You can find more information about using `.vsconfig` files [here](https://docs.microsoft.com/en-us/visualstudio/install/import-export-installation-configurations?view=vs-2022)
+2. Create a local directory for the repository, then run `git clone https://github.com/OData/odata.net.git` from the command prompt in that directory
+3. Open [odata.net/sln/OData.Tests.Performance.sln](OData.Tests.Performance.sln) using the previously installed Visual Studio
+4. In Visual Studio, right click `Solution 'OData.Tests.Performance' -> Build Solution`
+5. For each benchmark project, open the binary output folder in a command prompt and run its `exe` with the arguments `--filter *`. Ensure that the output doesn't not contain any failures or exit code errors. As an example, navigate to `C:\github\odata.net\test\PerformanceTests\ComponentTests\bin\Release\netcoreapp3.1` and run `Microsoft.OData.Performance.ComponentTests.exe --filter * > tests.txt 2>&1` and ensure that `tests.txt` does not contain any failures. NOTE: currently, there are no running benchmarks in `Microsoft.OData.Performance.ServiceTests`; this issue is currently being tracked [here](TODO).
```

Comment:
Note to self: don't merge PR until this work is actually tracked

Reply:
Did you create an ADO issue to track so you can close this comment?

Reply:
Thanks, I created an issue and I will consolidate with Clement if needed when he is back from vacation